src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -29,6 +32,61 @@ public FeedbackSubmissionEditPageData(AccountAttributes account, StudentAttribut
         isHeaderHidden = false;
     }
     
+    public void init(String regKey, String email, String courseId) {
+        String joinUrl = new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW).
+                                        withRegistrationKey(regKey).
+                                        withStudentEmail(email).
+                                        withCourseId(courseId).
+                                        toString();
```

Comment:
The `.` operator should be in front, shouldn't it (according to our style guide)?